.changeset/famous-hairs-compete.md

```diff
@@ -0,0 +1,7 @@
+---
+'@microsoft/atlas-css': minor
+'@microsoft/atlas-js': minor
+'@microsoft/atlas-site': minor
+---
+
+add tabs
```

Comment:
This gets compiled into a changelog. Could we attempt to write something a little less haphazard here?

Reply:
updated

Reply:
@AaronAstle Leaving this one open, as I've noticed that we only have a changeset for `atlas-site` - would you be able to to push up changesets for `atlas-css` and `atlas-js`?

Reply:
Oof, noticed that it's actually three changesets rolled into one. My mistake! I'll blame the limited preview from the main PR review screen.

That said, I think usually we prefer to keep these changesets as separate files anyways.